README.md

```diff
@@ -84,6 +81,10 @@ Q: Shouldn't we be putting new APIs in built-in modules?
 
 A: If built-in modules had come out before this gets to Stage 3, that sounds like a good option. However, so far the idea in TC39 has been not to block either thing on the other. Built-in modules still have some big questions to resolve, e.g., how/whether polyfills should interact with them.
 
+Q: Why is line breaking not included?
```

Comment:
This change is unrelated to the switch from segment iteration to boundary iteration.

Reply:
Yes, but it's a needed change.

Reply:
Edits not related to clarifying the iteration model should be done in a separate PR. Bundling things together like this complicates the history and makes it harder to retroactively observe changes.